app/services/analytics_events.rb

```diff
@@ -361,6 +361,33 @@ def forget_all_browsers_visited
     track_event('Forget All Browsers Visited')
   end
 
+  # @identity.idp.event_name Idv address submitted
+  # User submitted an idv address
+  def idv_address_submitted(
```

Comment:
can you document the params? Running `make lint_analytics_events` will help highlight these
```
app/services/analytics_events.rb:366 idv_address_submitted success (undocumented)
app/services/analytics_events.rb:366 idv_address_submitted errors (undocumented)
app/services/analytics_events.rb:366 idv_address_submitted address_edited (undocumented)
app/services/analytics_events.rb:366 idv_address_submitted error_details (undocumented)
make: *** [lint_analytics_events] Error 1
```

Reply:
yep, added now.